packages/flutter_adaptive_scaffold/CHANGELOG.md

```diff
@@ -58,4 +62,4 @@ Make fuchsia a mobile platform.
 
 ## 0.0.1
 
-* Initial release
+* Initial release
```

Comment:
What is the diff on this line? Is there a missing newline?

Reply:
yes, thx, fixed